great_expectations/expectations/metrics/map_metric_provider/map_condition_auxilliary_methods.py

```diff
@@ -198,14 +200,6 @@ def _pandas_map_condition_query(
                 domain_records_df[column_name].notnull()
             ]
 
-    elif "column_list" in accessor_domain_kwargs:
-        column_list: List[Union[str, sqlalchemy.quoted_name]] = accessor_domain_kwargs[
-            "column_list"
-        ]
-        verify_column_names_exist(
```

Comment:
This looks like a behavior change (deleting the verification that column names exist), should this maybe be in a separate PR?